packages/devtools-local-toolbox/src/components/Root.js

```diff
@@ -1,6 +1,8 @@
 const classnames = require("classnames");
 const { getValue, isDevelopment } = require("devtools-config");
 
+require("./Root.css");
```

Comment:
yeah, turns out that it wasn't being pulled in so the dark theme scrollbar looked pretty bad in windows.